doc/user/content/manage/blue-green.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Blue/Green Deployments"
+description: "How to perform Blue/Green Deployments in Materialize."
+menu:
+  main:
+    parent: manage
+    name: "Blue/Green Deployments"
+    weight: 11
+---
+
+Materialize offers some helpful tools to manage Blue/Green deployments. We recommend using a Blue/Green style workflow any time you need to deploy schema or view definition changes to production views.
+
+## Structuring your environment
+
+{{< note >}}
+We highly recommend using dbt to manage deployments, but this workflow can be recreated using your own custom scripts.
+{{</ note >}}
+
+1. Use schemas and clusters to isolate your changes. Schemas should be the primary container for database objects (views, indexes, materialized views), while clusters are the compute resources that will perform view maintenance and serve queries.
+2. Configure `profiles.yml` to set up the different targets. Use consistent naming across the clusters and schemas that will be deployed together.
```

Comment:
I'm a dbt noob, so it wasn't immediately obvious to me what `profiles.yml` was. Is it reasonable to add a blurb or sentence like, "Within your dbt project, configure `profiles.yml`..."